### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#643] [General]
Perhaps the biggest release so far!

- All objects in plugdata now have tooltips for inlets, outlets and the object itself. 
	- The documentation can also be accessed from the "reference" option on right-click
	- You can browse all object docs from the Object Browser, under the "+" button
	- big thanks to @tomara-x and @vasilymilovidov for making this happen!
- Cleanup of toolbar and a new main menu, with more comforts like "Recently Opened"
- New split view allows opening multiple patches next to eachother (thanks to @Nejrup)
- Improved DAW automation support
	- New panel with better UX
	- Params are now addressed by name instead of index
	- Works better in touch/latch more than before, generally more responsive
- Theming improvements
	- More GUI customisation
	- Support for having many themes instead of just two
	- Can now create, import and export themes
- Internal MIDI Synth: built-in General Midi synthesizer that can be selected as a midi output
	- Similar to AU DLS Synth that Max has on macOS, but cross-platform
- Grid improvements:
	- Can now use a real grid, or snap to objects, or both. 
	- Settable grid size
	- Grid now also applies to resizing objects
- Resizing objects is less glitchy (thanks @alcomposer)
- ELSE is update to 1.0-rc7, cyclone is updated to v0.7-0
- [else/bicoeff] and [else/messbox] work in plugdata now
- Better results and UX for autocompletion
- Heavy exporter now works on older versions of macOS
- Now has a CLAP plugin version
- You can now also address ELSE and cyclone objects with the "else/" and "cyclone/" prefix
- Many small improvements to GUI objects (thanks @alcomposer)
- Fixed a whole lot of bugs!!
- Support for unicode text in any currently spoken language in the world

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by gmaroulis** [comment]
This is really awesome work guys! I have one issue though and I cannot access the documentation at all. When I click on the help I am getting a message that the documentation cannot be found. Is this on purpose or a known issue ? Can I fix that on my end ? I am running plugdata 0.70 on macos 12.6. 

<img width="989" alt="Screenshot 2023-03-20 at 6 30 11 PM" src="https://user-images.githubusercontent.com/4683609/226406292-3efb3fc8-c331-4ee2-98ef-51fcee7140da.png">

**Reply by tomara-x** [reply]
go to your plugdata config folder (sorry i don't speak mac), it should have something like this
![Screenshot_2023-03-20_18-35-46](https://user-images.githubusercontent.com/86204514/226407507-0dc38ee6-11d5-4050-9597-78699c6e1293.png)
but yours is gonna have 0.7.0 instead, select that baby and delete it, then restart plugdata

**Comment by gmaroulis** [comment]
@tomara-x Thank you so much! I confirm that it worked for me!

**Reply by tomara-x** [reply]
great! have fun!
